test/e2e/broker_test.go

```diff
@@ -59,6 +59,12 @@ func TestBroker(t *testing.T) {
 	test.BrokerDelete(r, "foo4", true)
 	verifyBrokerNotfound(r, "foo3")
 	verifyBrokerNotfound(r, "foo4")
+
+	t.Log("create broker with class")
+	test.BrokerCreateWithClass(r, "foo5", "foo-class")
```

Comment:
Are there naming conventions for the class that we should be verifying… e.g., cannot start with numbers or symbols?

Reply:
There seems to be no server side validation for this so I am not sure.